MojioSDK/Models/User.swift

```diff
@@ -48,14 +48,13 @@ public class User : Object, Mappable {
         return self.Tags.toArray()
     }
 
-    public func json () -> String? {
+    public func jsonDict () -> NSDictionary {
         let dictionary : NSMutableDictionary = NSMutableDictionary()
         
         if self.UserName != nil {
             dictionary.setObject(self.UserName!, forKey: "UserName")
         }
-        if self.Emails.count > 0 {
-            self.email = (self.Emails.first! as Email).Address
+        if self.email != nil {
```

Comment:
Yeah. When I wrote the SDK, I was unfamiliar with if let. Will go through some of the old code at some point to change it over.